processor/src/main/java/org/mapstruct/ap/internal/processor/CdiComponentProcessor.java

```diff
@@ -48,4 +50,24 @@ protected boolean requiresGenerationOfDecoratorClass() {
     protected boolean additionalPublicEmptyConstructor() {
         return true;
     }
+
+    private Type getType(String simpleName) {
```

Comment:
`getCDIType`

Reply:
I didn't want to complicate things. This also fetches the `@Inject` from the inject package as well